csharp-basics/exercises/Arithmetic/Exercise2/Program.cs

```diff
@@ -0,0 +1,51 @@
+﻿class Program
+{
+    private static void Main(string[] args)
+    {
+		GetInput:
+        Console.WriteLine("Enter a number");
+        string userInput = Console.ReadLine();
+		(int, bool) numberAcquired = GetNumberFromInput(userInput);
```

Comment:
nosauc metodi par CollectNumbers();
tur iekšā arī nolasi Console.ReadLine()
un to var darīt ciklā ja tur nav skaitļa... 
nav vajadzība taisīt kko šitik nelasāmu un neuztveramu. :)